src/test/resources/com/microsoftopentechnologies/windowsazurestorage/configuration-as-code.yml

```diff
@@ -4,8 +4,9 @@ credentials:
       - credentials:
           - azureStorageAccount:
               blobEndpointURL: "https://blob.core.windows.net/"
+              cdnEndpointURL: "https://cdn-resource-name.azureedge.net/"
```

Comment:
this shouldn't be added in the default test, please add another one so we can make sure compatibility is maintained and that this isn't required by default

Reply:
Addressed in latest commit